src/RootWithIntl.js

```diff
@@ -53,6 +55,7 @@ class RootWithIntl extends React.Component {
                     <MainContainer>
                       <OverlayContainer />
                       <MainNav stripes={stripes} />
+                      <Handlers event={events.LOGIN} stripes={stripes} />
```

Comment:
Won't this fire on _every_ render, rather than only when the user has just logged in?